resources/assets/coffee/react/mp-history/game-header.coffee

```diff
@@ -0,0 +1,65 @@
+###
+# Copyright 2016 ppy Pty. Ltd.
+#
+# This file is part of osu!web. osu!web is distributed with the hope of
+# attracting more community contributions to the core ecosystem of osu!.
+#
+# osu!web is free software: you can redistribute it and/or modify
+# it under the terms of the Affero GNU General Public License version 3
+# as published by the Free Software Foundation.
+#
+# osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+{div, a, span, h1, h2} = React.DOM
+el = React.createElement
+
+class MPHistory.GameHeader extends React.Component
+  timeFormat: 'HH:mm:ss'
+
+  render: ->
+    timeStart = moment(@props.game.start_time).format @timeFormat
+    timeEnd = moment(@props.game.end_time).format @timeFormat
+
+    timeString = "#{timeStart} "
+
+    title = @props.beatmapset.title
+    title += " [#{@props.beatmap.version}]" if @props.beatmap.version
+
+    if @props.game.end_time
+      timeString += "- #{timeEnd}"
+    else
+      timeString += Lang.get 'multiplayer.match.in-progress'
+
+    a
+      className: 'mp-history-game__header'
+      href: (laroute.route 'beatmaps.show', beatmaps: @props.beatmap.id) if @props.beatmap.id
+      style:
+        backgroundImage: "url(#{@props.beatmapset.covers.cover})" if @props.beatmapset.covers.cover
```

Comment:
the condition is always true. A different check is needed (`cover_updated_at` being not null).